public/app/plugins/panel/canvas/editor/element/elementEditor.tsx

```diff
@@ -109,6 +110,14 @@ export function getElementEditor(opts: CanvasEditorOptions): NestedPanelOptions<
         });
       }
 
+      builder.addCustomEditor({
+        category: ['Data links'],
+        id: 'dataLinks',
+        path: 'links',
+        name: '',
+        editor: DataLinksEditor,
+      });
+
```

Comment:
I think it might make sense to have this editor be the last one vs in the middle (to better match current location of panel data links editor 🤔) what do you think?